index.html

```diff
@@ -84,5 +84,284 @@
+      </div>
+
+      <div class="skills">
+        <ul>
+          <li class="language">
+            <p class="fontweight500">Languages</p>
+            <div class="arrow">
+              <i class="fa-solid fa-angle-down"></i>
+            </div>
+          </li>
+          <li class="languages">
+            <ul>
+              <li class="writtenlanguage">
+                <img src="images/worksection/js.png" alt="javaScript" />
+                <p>JavaScript</p>
+              </li>
+
+              <li class="writtenlanguage">
+                <img src="images/worksection/html.png" alt="HTML" />
+                <p>HTML</p>
+              </li>
+
+              <li class="writtenlanguage">
+                <img src="images/worksection/css.png" alt="CSS" />
+                <p>CSS</p>
+              </li>
+            </ul>
+          </li>
+
+          <li class="frameworkdiv">
+            <ul>
+              <li class="frameworks">
+                <p class="fontweight500">Frameworks</p>
+                <div class="arrow">
+                  <i class="fa-solid fa-angle-right"></i>
+                </div>
+              </li>
+              <li class="line"></li>
+
+              <li class="frameworks">
+                <p>Skills</p>
+                <div class="arrow">
+                  <i class="fa-solid fa-angle-right"></i>
+                </div>
+              </li>
+              <li class="line"></li>
+            </ul>
+          </li>
+        </ul>
+      </div>
+    </section>
+
+    <!-- End of About me section -->
+
+    <!-- Content me section -->
+
+    <section class="contactme">
+      <h2 class="name col">Contact me</h2>
+      <p class="fontweight400 contactp">
+        If you have an application you are interested in developing, a feature that you need built
+        or a project that needs coding. I’d love to help with it
+      </p>
+
+      <img src="images/contact/rectangle4.png" alt="rectangle4" class="rectangle4contact" />
+      <img src="images/contact/ellipse3.png" alt="ellipse3" class="ellipse3contact" />
+      <img src="images/contact/ellipse12.png" alt="ellipse12" class="ellipse12contact" />
+      <img src="images/contact/ellipse5.png" alt="ellipse5" class="ellipse5contact" />
+      <img src="images/contact/rectangle7.png" alt="rectangle7" class="rectangle7contact" />
+
+      <form action="https://formspree.io/f/xbjwbpyj" method="POST" class="contactform">
+        <input
+          class="formradius forminput"
+          type="text"
+          placeholder="Your name"
+          maxlength="30"
+          required
+        />
+        <input class="formradius forminput" type="email" placeholder="Email" required />
+        <textarea
+          class="formradius"
+          name="message"
+          id="message"
+          cols="30"
+          rows="10"
+          maxlength="500"
+          required
+        >
+        Write your message here</textarea
```

Comment:
This is a bad practice, you should use place holder rather than this. 
This cause the "write your messages" to be seen as a value rather than a placeholder.
Also because you wanted to mimic it to look like a placeholder, you made the text faint which is a bad UI. 

Please use a placeholder attribute instead and correct the the text input value color